#ifndef _ICP_LOCALIZER_H
#define _ICP_LOCALIZER_H

#include <string>
#include "commons.h"
#include <pcl/io/pcd_io.h>
#include <pcl/filters/voxel_grid.h>
#include <pcl/filters/passthrough.h>
#include <pcl/registration/icp.h>
#include <pcl/features/normal_3d.h>

namespace IG_LIO
{
    
} // namespace IG_LIO


#endif // _ICP_LOCALIZER_H